import-export-cli/box/resources/sample/api_params.yaml

```diff
@@ -0,0 +1,20 @@
+environments:
+    - name:
+      configs:
+        endpoints:
+          production:
+          sandbox:
+        certs:
+          - hostName:
+            alias:
+            path:
+          - hostName:
+            alias:
+            path:
+        mutualSslCerts:
+          - tierName:
+            alias:
+            path:
+          - tierName:
+            alias:
+            path:
```

Comment:
Shall we add a line at the eof?

Reply:
Fixed with f75a26712774c479a96a139dc5fdc95d089bb104